cmd/start.go

```diff
@@ -38,7 +38,7 @@ func startCmd(a *appState) *cobra.Command {
 		Use:     "start path_name",
 		Aliases: []string{"st"},
 		Short:   "Start the listening relayer on a given path",
-		Args:    withUsage(cobra.MinimumNArgs(1)),
+		Args:    withUsage(cobra.MinimumNArgs(0)),
 		Example: strings.TrimSpace(fmt.Sprintf(`
```

Comment:
Examples could be updated here.

Reply:
Fixed examples!! Thank you @mark-rushakoff